framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -170,7 +170,7 @@ int esm_set_device_property(char *name, const wifi_mode_e *mode, int ea_mode, co
 	}
 
 	if (name == NULL) {
-		name = def_device_name;
+		name = DEF_DEVICE_NAME;
```

Comment:
is it working that assigning MACRO to variable?

Reply:
It is working. I will assign it using "strncpy".